streams/src/main/java/org/apache/kafka/streams/kstream/Windows.java

```diff
@@ -39,10 +51,12 @@ protected Windows() {
     }
 
     /**
-     * Set the window maintain duration in milliseconds of streams time.
+     * Set the window maintain duration (retention time) in milliseconds.
```

Comment:
Why do we need "duration" or "maintain" ? Can't we say just ...the window retention time...